openpathsampling/snapshot.py

```diff
@@ -29,8 +30,8 @@ class Configuration(object):
     engine = None
     load_lazy = True
 
-    def __init__(self, coordinates=None, box_vectors=None,
-                 potential_energy=None, topology=None):
+    def __init__(self, coordinates = None, box_vectors = None,
+                 potential_energy = None, topology = None):
```

Comment:
I think I changed the default settings in my IDE at some point but changed it back to PEP and forgot to check everywhere :)